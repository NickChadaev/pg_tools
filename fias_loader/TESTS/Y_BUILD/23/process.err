ЗАМЕЧАНИЕ:  индекс "ie4_as_addr_obj" не существует, пропускается
ЗАМЕЧАНИЕ:  отношение "__adr_area_fias" уже существует, пропускается
ЗАМЕЧАНИЕ:  отношение "__adr_area_fias" уже существует, пропускается
