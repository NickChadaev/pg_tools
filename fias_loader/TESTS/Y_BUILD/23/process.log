2023-11-07 18:38:55 ./
127.0.0.1 5434 unsi_test_23 postgres stage_gar_c_01.csv 2023-10-26
--------------------------------------------------------
[18:38:55] ... Start process
[18:38:55] ...  -- Version
[18:38:55] ...  -- Очистка от данных
[18:38:58] ...  -- Убираем индексы
[18:38:58] ...  -- Set UNLOGGED
[18:39:03] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_ADDHOUSE_TYPES_20231026_1b8ca463-4b89-4a94-96e9-2567cd8bae55.XML
[18:39:04] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_ADDR_OBJ_TYPES_20231026_f94d64d9-84f7-4717-8c8c-38f9f18a8f85.XML
[18:39:04] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_APARTMENT_TYPES_20231026_b7af29ce-b522-4733-8996-50a115c9ee6c.XML
[18:39:04] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_HOUSE_TYPES_20231026_766c4e6c-f5bf-416c-b939-76e5d7169016.XML
[18:39:04] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_NORMATIVE_DOCS_KINDS_20231026_247eb17b-646d-4dba-9170-21f6e5572570.XML
[18:39:04] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_NORMATIVE_DOCS_TYPES_20231026_ec4bc25a-31a1-4ec8-805b-eacabe781aab.XML
[18:39:04] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_OBJECT_LEVELS_20231026_10c61fea-632f-4017-91ab-200ed516b142.XML
[18:39:04] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_OPERATION_TYPES_20231026_bf3006f0-f7c5-4e29-aadb-3671d5b3fe59.XML
[18:39:04] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_PARAM_TYPES_20231026_88d99e07-ce03-41e4-9288-f1ebfaa80c26.XML
[18:39:04] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_ROOM_TYPES_20231026_a8f37b4d-bd22-4f73-a8a8-f489efdc4f2b.XML
[18:39:04] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_ADDR_OBJ_20231026_90c4772d-8e23-47e3-8d3c-70a219103f6d.XML
[18:39:43] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_ADDR_OBJ_DIVISION_20231026_453b21e8-e788-4aa0-a602-5501464a3f70.XML
[18:39:43] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_ADDR_OBJ_PARAMS_20231026_a0603a44-3da4-4119-ac7e-503e570da5f4.XML
[18:43:44] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_ADM_HIERARCHY_20231026_8e71ec34-445a-481c-ae70-ba33e16523ca.XML
[19:17:43] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_APARTMENTS_20231026_c7dbddb2-4789-4724-8c46-80efa5610e98.XML
[19:29:12] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_APARTMENTS_PARAMS_20231026_69e286ee-4f4f-42d9-a994-3f85b9c7e810.XML
[19:49:55] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_CARPLACES_20231026_803868ef-50f8-4cd1-b1a7-7d8280fc8f5d.XML
[19:50:13] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_CARPLACES_PARAMS_20231026_0ae9ccec-3c93-48a4-84f3-05690cb58662.XML
[19:50:14] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_CHANGE_HISTORY_20231026_783fecf5-bde5-4e1c-ac6c-7eba0bbc7562.XML
[20:53:40] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_HOUSES_20231026_ad35c4dc-de8c-4e56-a97c-a67aad46f44d.XML
[21:22:25] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_HOUSES_PARAMS_20231026_e37c6734-7d4c-446e-9cb0-e817ccaf58bf.XML
[23:20:13] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_MUN_HIERARCHY_20231026_39bab685-0e90-405c-b432-17cb1957e6c7.XML
[23:48:29] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_NORMATIVE_DOCS_20231026_d5306e8e-5c31-4c76-bd37-5bbb27d570b2.XML
[23:54:03] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_REESTR_OBJECTS_20231026_53da8c10-522d-496c-8d72-0702739c066c.XML
[00:14:29] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_ROOMS_20231026_7ec23618-d24e-4824-9847-15a04cee1cd8.XML
[00:14:31] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_ROOMS_PARAMS_20231026_9700fa0a-f70e-416a-a187-7bfe9afc1e07.XML
[00:14:33] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_STEADS_20231026_b45e0d52-f960-42d5-a1b5-1d82c6635d9b.XML
[00:19:59] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/23/AS_STEADS_PARAMS_20231026_d66aa4a5-d4a1-484a-9431-bfe15a801e71.XML
[01:05:37] ...  -- Data finish
[01:05:38] ...  -- Set LOGGED
[01:20:10] ... Stop process
--------------------------------------------------------
2023-11-08 10:53:04 ./
127.0.0.1 5434 unsi_test_23 postgres stage_3_01.csv 2023-10-26
--------------------------------------------------------
[10:53:04] ... Start process, Подготовка
[10:53:04] ... -- Проверка региона
[10:53:04] ...  -- Set LOGGED
[10:53:04] ...  -- Установка индексного покрытия в схеме в схеме GAR_FIAS
[10:57:42] ...  -- Очистка данных во временной схеме
[10:58:43] ... -- Заполнение таблицы с дефектными данными
[10:59:28] ...  -- Обработка таблицы с дефектными данными
[10:59:29] ...  -- Ещё раз проверка дублей
[10:59:33] ... Stop process
--------------------------------------------------------
2023-11-08 11:04:18 ./
127.0.0.1 5434 unsi_test_23 postgres stage_31_01.csv 2023-10-26
--------------------------------------------------------
[11:04:18] ... Start process, Подготовка
[11:04:18] ... -- Проверка региона
[11:04:19] ...  Установка gar_tmp.obj_seq/gar_tmp.obj_hist_seq
[11:04:19] ...  -- Агрегация параметров
[11:06:44] ...  -- Агрегация адресных пространств
[11:07:55] ...  -- Агрегация домовладений
[11:08:58] ...  -- Заполнение рабочей таблицы "gar_tmp.xxx_obj_fias"
[11:10:42] ... Stop process
--------------------------------------------------------
2023-11-09 19:38:37 ./
127.0.0.1 5434 unsi_test_23 postgres stage_3_23.csv 2023-11-09
--------------------------------------------------------
[19:38:37] ... Start process, Подготовка
[19:38:37] ... -- Проверка региона
[19:38:37] ...  -- Set LOGGED
[19:38:37] ...  -- Установка индексного покрытия в схеме в схеме GAR_FIAS
[19:40:31] ...  -- Очистка данных во временной схеме
[19:41:56] ... -- Заполнение таблицы с дефектными данными
[19:42:05] ...  -- Обработка таблицы с дефектными данными
[19:42:07] ...  -- Ещё раз проверка дублей
[19:42:10] ... Stop process
--------------------------------------------------------
2023-11-09 19:43:19 ./
127.0.0.1 5434 unsi_test_23 postgres stage_31_23.csv 2023-11-09
--------------------------------------------------------
[19:43:19] ... Start process, Подготовка
[19:43:19] ... -- Проверка региона
[19:43:19] ...  -- Очистка адресных пространств
[19:43:21] ...  -- Очистка  домовладений
[19:43:21] ...  -- Очистка агрегатов
[19:43:23] ...  Установка gar_tmp.obj_seq/gar_tmp.obj_hist_seq
[19:43:24] ...  -- Агрегация параметров
[19:45:59] ...  -- Агрегация адресных пространств
[19:47:26] ...  -- Агрегация домовладений
[19:48:58] ...  -- Заполнение рабочей таблицы "gar_tmp.xxx_obj_fias"
[19:50:40] ... Stop process
--------------------------------------------------------
