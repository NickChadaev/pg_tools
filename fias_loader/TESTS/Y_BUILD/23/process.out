 save_gar_version 
------------------
                9
(1 строка)

CALL
CALL
CALL
